react/features/recording/components/LiveStream/LiveStreamButton.web.js

```diff
@@ -85,17 +84,13 @@ class LiveStreamButton extends AbstractLiveStreamButton<Props> {
  */
 function _mapStateToProps(state: Object, ownProps: Props) {
     const abstractProps = _abstractMapStateToProps(state, ownProps);
-    const localParticipant = getLocalParticipant(state);
-    const { features = {} } = localParticipant;
     let { visible } = ownProps;
 
-    let _disabled = false;
+    const _disabled = abstractProps.disabledByFeatures;
     let _liveStreamDisabledTooltipKey;
 
     if (!abstractProps.visible
-            && String(features.livestreaming) !== 'disabled') {
-        _disabled = true;
-
+        && _disabled !== undefined && !_disabled) {
```

Comment:
I think `_disabled !== undefined && !_disabled`  might be redundant checks. ` !_disabled` is already checking `_disabled !== undefined`.

Reply:
Also, can you explain the intent of this check to me? If a button is not visible, but it's disabled, you want it to be visible?